scripts/image_pi

```diff
@@ -86,7 +97,7 @@ est_time=$(echo "$img_size_mb/$write_speed_mb/60+0.5" | bc -l | cut -d'.' -f1)
 echo "Copying the image to the AmpliPi. This takes about $est_time minutes."
 echo "Go get a coffee or something :)"
 echo "Started at: $(date)"
-sudo dd if=$img_file of=$diskpath bs=4MiB status=progress
+sudo dd if=$img_file of=$diskpath oflag=sync bs=1M status=progress
```

Comment:
Ok, I finally was able to reproduce the problem that this PR fixes with the "remove access to the file while waiting for bootup" as mentioned above. Since avoiding the global `set -e` was suggested, here's my suggestion (with removing the `trap handle_error ERR` and `set -e` lines above.
```suggestion
sudo dd if=$img_file of=$diskpath oflag=sync bs=1M status=progress || handle_error
```